### PR TITLE
Update README for docs

### PR DESCRIPTION
**Pull request by ChumpChief**
In particular, adding a section elaborating on the stages the API docs go through when building to help with debugging doc issues.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -17,44 +11,87 @@ cd docs
...
 ```
 
-Open `http://localhost:1313` to preview the site. Any changes to the source
-content will automatically force an HTML re-render allowing you to preview your
-modifications in quasi real time.
+## Building the documentation
 
-If you want to debug the generated content, you can build the site and see the
-output in the `public/` folder:
+Run the `build` script to build the site. The output will be in the `public/` folder.
```
Comment (heliocliu) [inline docs/README.md:33]: I think it might be good to clarify here if when you build the site if it affects the site preview from npm start

Comment (ChumpChief) [reply docs/README.md:33]: I wasn't aware - what does it affect?

Comment (tylerbutler) [reply docs/README.md:33]: Build and start are not related. Start serves from an in-memory dev server and doesn't serve anything from the public folder.

Comment (tylerbutler) [reply docs/README.md:33]: That said, there are some tasks that are only done in the build script for performance reasons. That's worth calling out.

**Review by tylerbutler — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (ChumpChief) [reply docs/README.md:33]: I think I'll omit for now, the purpose of this change is to add info on how to debug bad doc output.  Details on motivations for particular script decisions could be a separate change.

**Pull request merged**